roles/docker/tasks/configure.yml

```diff
@@ -17,6 +17,10 @@
 - name: Reset ssh connection to allow user changes to affect current user
   meta: reset_connection
 
+- name: reload systemd
+  command: systemctl daemon-reload
+  when: ansible_distribution == 'Ubuntu' and ansible_lsb.major_release|int >= 20
```

Comment:
Is there a reason why you had chosen to do `ansible_lsb.major_release` instead of doing `ansible_facts['distribution']`. I don't think I have seen the former before, and was curious what the difference was?

Reply:
Sorry, I somehow missed that review. Not intended, I just googled and found those variables.
I changed that to use the facts, which seems to be working the same way:
https://docs.ansible.com/ansible/latest/user_guide/playbooks_conditionals.html

Those are also available with older versions of Ansible (I checked until 2.5).

Reply:
Cool, no problem. Just was curious, awesome I will merge this now :slightly_smiling_face: